### PR TITLE
[Product] Change product controller

### PR DESCRIPTION
**Pull request by alvinmatthew12**
# What
As the title say

# How
Add comment

# Why
Want to test PR

### CODE REVIEW

File: app/Http/Controllers/v1/Admin/ProductController.php [hunk 1]
```diff
@@ -13,6 +13,11 @@ class ProductController extends Controller
 {
     public function index()
     {
+        // changes here to get all products
+        // comment bugs here
+        // bugs
+        // bugs
+        // bugs
```
Comment (alvinmatthew12) [inline app/Http/Controllers/v1/Admin/ProductController.php]: Please fix bugs

File: app/Http/Controllers/v1/Admin/ProductController.php [hunk 2]
```diff
@@ -13,6 +13,11 @@ class ProductController extends Controller
 {
     public function index()
     {
+        // changes here to get all products
+        // comment bugs here
+        // bugs
```
Comment (cindyadonia) [inline app/Http/Controllers/v1/Admin/ProductController.php]: Please remove this

File: app/Http/Controllers/v1/Admin/ProductController.php [hunk 3]
```diff
@@ -13,6 +13,11 @@ class ProductController extends Controller
 {
     public function index()
     {
+        // changes here to get all products
```
Comment (kisusyenni) [inline app/Http/Controllers/v1/Admin/ProductController.php:16]: //test

**Review by kisusyenni — DISMISSED** [review]

**Review by alvinmatthew12 — COMMENTED** [review]
Please make the PR description more clear

**Review by kisusyenni — APPROVED** [review]

**Review by cindyadonia — APPROVED** [review]

**Pull request merged**